stac_pydantic/api/search.py

```diff
@@ -102,30 +98,38 @@ def validate_bbox(cls, v: BBox) -> BBox:
 
     @field_validator("datetime")
     @classmethod
-    def validate_datetime(cls, v: str) -> str:
-        if "/" in v:
-            values = v.split("/")
-        else:
-            # Single date is interpreted as end date
-            values = ["..", v]
-
-        dates: List[dt] = []
-        for value in values:
-            if value == ".." or value == "":
-                continue
-
-            dates.append(parse_rfc3339(value))
-
+    def validate_datetime(cls, value: str) -> str:
+        # Split on "/" and replace no value or ".." with None
+        values = [v if v and v != ".." else None for v in value.split("/")]
+        # If there are more than 2 dates, it's invalid
         if len(values) > 2:
-            raise ValueError("Invalid datetime range, must match format (begin_date, end_date)")
-
-        if not {"..", ""}.intersection(set(values)):
-            if dates[0] > dates[1]:
-                raise ValueError(
-                    "Invalid datetime range, must match format (begin_date, end_date)"
-                )
-
-        return v
+            raise ValueError(
+                "Invalid datetime range. Too many values. Must match format: {begin_date}/{end_date}"
+            )
+        # If there is only one date, insert a None for the start date
+        if len(values) == 1:
+            values.insert(0, None)
```

Comment:
I ran into this MR after I created #145. I see that the above matches the current behavior, nice. 

But I also think the start and end should be the same if a single value is given. (Otherwise one cannot tell the difference between a single value, and a open-ended range, without looking at the original `datetime` again.) So I guess this could be changed to:

```
# If there is only one date, duplicate to use for both start and end dates
if len(values) == 1:
    values.insert(0, values[0])
``` 

Or maybe easier to read then:

```
    values.append(values[0])
```